driver-core/src/main/com/mongodb/client/model/search/FuzzySearchOptions.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.client.model.search;
+
+import com.mongodb.annotations.Evolving;
+import org.bson.conversions.Bson;
+
+/**
+ * Fuzzy search options that may be used with some {@link SearchOperator}s.
+ *
+ * @mongodb.atlas.manual atlas-search/autocomplete/ autocomplete operator
```

Comment:
Why the link to autocomplete?

Reply:
It's because the `autocomplete` operator also supports fuzzy search with the same syntax. I.e., this class is relevant for both `text` and `autocomplete` operators.